Otec/NewspaperSubscriptionTableViewCell.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  NewspaperSubscriptionTableViewCell.swift
+//  Otec
+//
+//  Created by El gera de la gente on 6/13/16.
+//  Copyright © 2016 Inaka. All rights reserved.
+//
+
+import UIKit
+
+class NewspaperSubscriptionTableViewCell: UITableViewCell {
+    @IBOutlet weak var newspaperNameLabel: UILabel!
+    @IBOutlet weak var newspaperDescriptionLabel: UILabel!
+
+    var newspaper : Newspaper! {
+        didSet {
+            self.newspaperNameLabel.text = newspaper.id
+            self.newspaperDescriptionLabel.text = newspaper.description
```

Comment:
Hmmm... If `description` property is something custom (something we have defined, not native) then we should name it something else; otherwise, it could conflict with the `description` var from `CustomStringConvertible` protocol.
This applies to all our models in all of our projects ⚠️